pkg/common/morpc/codec.go

```diff
@@ -30,6 +30,8 @@ var (
 	flagPayloadMessage byte = 1
 	flagWithChecksum   byte = 2
 	flagCustomHeader   byte = 4
+
+	defaultMaxMessageSize = 1024 * 1024 * 100
```

Comment:
Why do you need two?

Reply:
it will only use morpc's default max message body size